libmemcached will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libmemcached it will fail in the following way:

```
2024/11/12 23:12:17 INFO running step "test/pkgconf"
2024/11/12 23:12:17 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:17 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:17 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:17 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:17 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + basename /home/build/melange-out/libmemcached uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + dev_pkg=libmemcached uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + apk info -L libmemcached uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + grep -q ^Name: usr/lib64/pkgconfig/libmemcached.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + basename usr/lib64/pkgconfig/libmemcached.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + lib_name=libmemcached uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + echo usr/lib64/pkgconfig/libmemcached.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 WARN + grep -q '^usr/lib/pkgconfig/libmemcached.pc$\|^usr/share/pkgconfig/libmemcached.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:17 INFO pod 6e963cf314855e46f72677995e7f14bae577ef3a39ac2c7d4c7042b226008fbc terminated
2024/11/12 23:12:17 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:17 INFO   workspace dir: /home/user/tmp/melange-workspace-2041880544
2024/11/12 23:12:17 INFO   guest dir: /home/user/tmp/melange-guest-1890791114
2024/11/12 23:12:17 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libmemcached] Error 1
```

--- a/libmemcached.yaml
+++ b/libmemcached.yaml
@@ -1,7 +1,7 @@
 package:
   name: libmemcached
   version: 1.1.4
-  epoch: 1
+  epoch: 2
   description: Client library and command line tools for memcached server (resurrected)
   copyright:
     - license: BSD-3-Clause
@@ -41,6 +41,8 @@ pipeline:
       DESTDIR="${{targets.destdir}}" cmake --install build
       # just junk top-level readme/changelog files from repo
       rm -r "${{targets.destdir}}"/usr/share/doc
+      mkdir -p "${{targets.destdir}}"/usr/lib/pkgconfig/
+      mv "${{targets.destdir}}"/usr/lib64/pkgconfig/libmemcached.pc "${{targets.destdir}}"/usr/lib/pkgconfig/libmemcached.pc
 
   - uses: strip
 
@@ -48,6 +50,9 @@ subpackages:
   - name: libmemcached-dev
     pipeline:
       - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
     dependencies:
       runtime:
         - libmemcached

--- a/libmemcached.yaml
+++ b/libmemcached.yaml
@@ -32,6 +32,7 @@ pipeline:
   - runs: |
       cmake -B build -G Ninja \
       	-DCMAKE_BUILD_TYPE=None \
+      	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
       	-DCMAKE_INSTALL_PREFIX=/usr \
       	-DBUILD_SHARED_LIBS=ON \
       	-DENABLE_SASL=ON
@@ -41,8 +42,6 @@ pipeline:
       DESTDIR="${{targets.destdir}}" cmake --install build
       # just junk top-level readme/changelog files from repo
       rm -r "${{targets.destdir}}"/usr/share/doc
-      mkdir -p "${{targets.destdir}}"/usr/lib/pkgconfig/
-      mv "${{targets.destdir}}"/usr/lib64/pkgconfig/libmemcached.pc "${{targets.destdir}}"/usr/lib/pkgconfig/libmemcached.pc
 
   - uses: strip
 

